sdks/python/apache_beam/yaml/yaml_io.py

```diff
@@ -152,10 +152,11 @@ def _create_parser(
   elif format == 'avro':
     beam_schema = avroio.avro_schema_to_beam_schema(schema)
     covert_to_row = avroio.avro_dict_to_beam_row(schema, beam_schema)
+    # pylint: disable=line-too-long
```

Comment:
Can we break the below line and remove this?

```
return (
        beam_schema,
        lambda record: covert_to_row(
            fastavro.schemaless_reader(  # type: ignore[call-arg]
                    io.BytesIO(record), schema)))  
```

Reply:
I'm not sure that would actually ignore the check on the appropriate line with the split, I'm inclined to leave this as-is since it's _ideally_ a temporary change